apps/mobile-wallet/src/components/layout/ScrollScreen.tsx

```diff
@@ -181,7 +182,7 @@ const ScrollViewContainer = styled.View`
 `
 
 const BottomButtonsContainer = styled.View`
-  margin: 0 ${DEFAULT_MARGIN}px;
+  margin: ${Platform.OS === 'ios' ? 0 : undefined} ${DEFAULT_MARGIN}px;
```

Comment:
On Android this is gonna be `margin: undefined XXpx`. This doesn't cause any issue?

Reply:
it does not